src/odemis/acq/fastem.py

```diff
@@ -452,6 +454,8 @@ def acquire_roa(self, dataflow):
             expected_time = len(self._fields_remaining) * total_field_time
             self._future.set_progress(start=time.time(), end=time.time() + expected_time)
 
+        logging.debug("Finished acquiring all fields of ROA.")
```

Comment:
This one seems to be redundant to the log in line 410.

Reply:
I wanted to have one to be sure that all fields were acquired successfully, the one on line 410 is also logged when the acquisition was canceled or an exception was raised. I could update this one to `"Successfully acquired all fields of ROA."`

Reply:
Yes I think this one is nice.